Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
+| `IfcBoundingBox`                      | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBoxedHalfSpace`                   | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcBSplineCurveWithKnots`            | :heavy_check_mark:   | `SplineConverter.`         | -                     | -     |
+| `IfcCartesianPoint`                   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcCartesianPointList2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCartesianPointList3D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCartesianTransformationOperator2D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator3D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
+| `IfcCompositeCurveSegment`            | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcCurveSegment`                     | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcDirection`                        | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcEllipse`                          | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcExtrudedAreaSolid`                | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcExtrudedAreaSolidTapered`         | :x:                  | -                          | -                     | -     |
+| `IfcFaceBasedSurfaceModel`            | :heavy_check_mark:   | `FaceConverter`            | -                     | -     |
+| `IfcFacetedBrep`                      | :x:                  | `SolidModelConverter`      | -                     | -     |
```

Comment:
I'm almost positive, that we support this.

Reply:
Resolved 31a5bd4